packages/forms/backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/ContainerBorderStyle.java

```diff
@@ -32,10 +32,24 @@ public final class ContainerBorderStyle {
 
     private ContainerBorderLineStyle lineStyle;
 
+    private String margin;
+
+    private String padding;
+
+    private String gap;
+
+    private ContainerContentAlignment justifyContent;
+
+    private ContainerContentAlignment alignItems;
```

Comment:
I don't see why those properties are related to a container "border". I don't know why this code should be there.

Reply:
properties are not link to border anymore